apple/internal/apple_framework_import.bzl

```diff
@@ -353,6 +353,13 @@ target.
                 [apple_common.Objc, AppleFrameworkImportInfo],
             ],
         ),
+        "avoid_link": attr.bool(
```

Comment:
Is there precedent to the `avoid_link` name? If not, maybe a name that indicates it's for bundling?

Reply:
nope I totally made this up, any thoughts on a specific replacement?

Reply:
at first I thought `bundle_only`, but then that has the problem of not mentioning linking. Then I thought `unlinked_bundle`. It seems not great but maybe ok.